sources/packages/api/src/utilities/constants.ts

```diff
@@ -2,3 +2,7 @@
 export const EXTENDED_DATE_FORMAT = "MMMM, D YYYY";
 export const PIR_DENIED_REASON_OTHER_ID = 1;
 export const COE_WINDOW = 21;
+export const COE_DENIED_REASON_OTHER_ID = 1;
+export const COE_REQUEST_NOT_FOUND_ERROR = "COE_REQUEST_NOT_FOUND_ERROR";
```

Comment:
I would recommend move COE_DENIED_REASON_OTHER_ID and COE_REQUEST_NOT_FOUND_ERROR to the service class. I gave some further explanations on why in the previous comment about the same constant. Always open to discussion.

Reply:
IMO. We can functionally/technically create multiple constant files and mange them. eg: application-constants.ts, coe-constants.ts. Just a suggestion.

Reply:
the idea was to add all constants in the utils/constants file. right @guru-aot ? yes, @dheepak-aot  suggestions also sound good, if we predict we will have more constants in our application

Reply:
It does not seem right to me. If there is a constant that is globally used and it is generic, maybe, if there is not a specific file to add, we can add to a common/generic/altogether constants file, otherwise, why not create the constants near the folder/context that they belong to?
Following the same logic, are we going to add all the enums in the same file?
We also have database constants inside the database folder (because they belong to that context), do we need to move these constants inside the "common/generic/altogether constants file"?